.github/workflows/validate.yml

```diff
@@ -37,10 +37,6 @@ jobs:
     steps:
       - name: Check out code
         uses: actions/checkout@v2
-      - name: Install Go
```

Comment:
I am a bit confused about this removal. Then is the version of go preinstalled in the Ubuntu 20.04 runner used?  Or the version depended from the golangci-lint action?

Reply:
I removed by mistake. However it's strange as the validate action just passed. Should I revert this change?

Reply:
I suspect it's because the default go installation in the ubuntu is used for the workflow. The official golangci-lint action readme suggests putting `actions/setup-go` before the step, so I think it's safer to revert it.
https://github.com/golangci/golangci-lint-action#how-to-use